extensions/tree-inspector/tree-inspector.js

```diff
@@ -399,6 +413,23 @@ var TreeInspector = (function() {
         updateCheckboxControl.appendChild(createElement("label", "Update When Debugger Breaks", {
             "for": id("RefreshOnBreak")
         }));
+        this.refreshOnBreakAndContinueGroupElement = createElement("span", null, { 

```

Comment:
UI nit: Can we add a little bit more spacing between "Update When Debugger Breaks" and the checkbox for "Continue Execution"? They seem a little too close to each other.